source/Octopus.Tentacle.Kubernetes.Tests.Integration/Setup/KubernetesClusterInstaller.cs

```diff
@@ -31,7 +31,7 @@ public KubernetesClusterInstaller(TemporaryDirectory tempDirectory, string kindE
         this.kubeCtlPath = kubeCtlPath;
         this.logger = logger;
 
-        clusterName = $"tentacleint-{DateTime.Now:yyyyMMddhhmmss}";
+        clusterName = $"tentaclient-{DateTime.Now:yyyyMMddhhmmss}";
```

Comment:
?

Reply:
hehe - I kept reading "tentacle-cleint" - but now I see its tentacle_int - changing it back!